src/builtin_plugins.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import Plugins, {Plugin} from './plugins'
+import path from 'path'
+
+export default class BuiltinPlugins {
+  constructor (plugins: Plugins) {
+    this.plugins = plugins
+  }
+
+  plugins: Plugins
+
+  /**
+   * list user plugins
+   * @returns {Plugin[]}
+   */
+  list (): Plugin[] {
+    let commandsPath = path.resolve(path.join(__dirname, 'commands'))
+    return [new Plugin('builtin', commandsPath, this.plugins)]
```

Comment:
should certainly go in a different PR, but thoughts on making this an options instead?

```js
return [new Plugin({type: 'builtin', path: commandsPath, plugins: this.plugins})]
```